applications/CableNetApplication/custom_python/cable_net_python_application.cpp

```diff
@@ -0,0 +1,55 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Klaus B. Sautter
+//
+
+
+// System includes
+
+#if defined(KRATOS_PYTHON)
+// External includes
+#include <pybind11/pybind11.h>
+
+
+// Project includes
+#include "includes/define.h"
```

Comment:
```suggestion
#include "includes/define_python.h"
```

this was wrong in the app-generator